### ISSUE TITLE
Cannot add settings via JSON for flake8 plugins

### ISSUE THREAD

**Issue by triplmint** [microsoft/vscode-flake8#137]
Hi!
I'm trying to use the setting `--docstring-convention=numpy` from the [flake8-docstrings](https://pypi.org/project/flake8-docstrings/) plugin in my settings.json file as such:
```
{
    "flake8.importStrategy": "fromEnvironment",
    "flake8.args": [
        "--docstring-convention=numpy",
    ],
}
```

In the extension output, I get:
```
2023-03-29 11:33:20.927 [info] usage: flake8 [options] file file ...
flake8: error: unrecognized arguments: --docstring-convention=numpy
```

I have flake8-docstrings installed alongside flake8 and confirmed that this flag works in the integrated terminal.

Thanks for your help!

**Comment by karthiknadig** [comment]
@triplmint Can you share the logs from Output > flake8 ?

**Comment by karthiknadig** [comment]
@triplmint Currently with `conda` you may have to start VS Code like this to get the best experience:
```
> cd /Users/triplmint/dev/linting-templates
> conda activate -n main
> code .
```

Alternatively, you can also try setting the `flake8.interpreter` to `["conda", "run", "--live-stream", "-n", "main", "python"]`.

We are working on making this experience better, this is a temporary workaround.

**Comment by triplmint** [comment]
Still ending up with `flake8: error: unrecognized arguments: --docstring-convention=numpy` with either suggestion, but no worries! I can just use `pydocstyle` and live with it just fine until conda support is improved. Thanks! Would you like for me to leave this issue open or close it for now?

**Comment by karthiknadig** [comment]
@triplmint Can you do one more ting just to make sure? please run this:
```
conda run --live-stream -n main python -m pip list
```
That should list the `flake8-docstrings` package.

**Comment by triplmint** [comment]
Output from the conda command:
```
flake8                  6.0.0
flake8-docstrings       1.7.0
```
among other things.

**Comment by karthiknadig** [comment]
@triplmint I am really not sure why this is causing an error in your case. This is what I see, with conda environment.

```
// BEFORE flake8-docstrings install
2023-03-31 11:36:57.339 [info] c:\Users\kanadig\.conda\envs\flaketest\python.exe -m flake8 --format='%(row)d,%(col)d,%(code).1s,%(code)s:%(text)s' --docstring-convention=numpy c:\GIT\demo\toml_req_test\myscript.py
2023-03-31 11:36:57.339 [info] CWD Linter: c:\GIT\demo\toml_req_test
2023-03-31 11:36:57.375 [info] usage: flake8 [options] file file ...
flake8: error: unrecognized arguments: --docstring-convention=numpy

// AFTER flake8-docstrings install
2023-03-31 11:38:10.534 [info] c:\Users\kanadig\.conda\envs\flaketest\python.exe -m flake8 --format='%(row)d,%(col)d,%(code).1s,%(code)s:%(text)s' --docstring-convention=numpy c:\GIT\demo\toml_req_test\myscript.py
2023-03-31 11:38:10.535 [info] CWD Linter: c:\GIT\demo\toml_req_test
2023-03-31 11:38:11.150 [info] file:///c%3A/GIT/demo/toml_req_test/myscript.py :
'1,1,D,D100:Missing docstring in public module'
'13,11,F,F821:undefined name 'x''

```

Not really sure why this is not getting picked up in your case.

**Comment by triplmint** [comment]
Got it working @karthiknadig! i had installed into my conda repo using pip, installed from conda-forge and the interpreter is picking it up. thanks!
